### PR TITLE
[User manager] User schema and user data's CRUD based on k8s secret

### PR DESCRIPTION
**Pull request by ydye**

### CODE REVIEW

File: src/rest-server/src/util/manager/user/crudK8sSecret.js [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+    const token = readFileSync(option.k8sAPIServerTokenFile, 'ascii');
+    config.reqeustConfig.headers = {Authorization: `Bearer ${token}`};
+  }
+}
+
+function getSecretRootUri(config) {
+  return `${config.namespace}/secrets`;
+}
+
+async function read(key, config) {
```
Comment (sunqinzheng) [inline src/rest-server/src/util/manager/user/crudK8sSecret.js]: need some documentation comments about the config param

File: src/rest-server/src/util/manager/user/crudK8sSecret.js [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+const User = require('./user');
+const axios = require('axios');
+const {readFileSync} = require('fs');
+const {Agent} = require('https');
+
+function initConfig(apiServerUri, namespace, option) {
```
Comment (sunqinzheng) [inline src/rest-server/src/util/manager/user/crudK8sSecret.js]: I think the user secret's namespace should not be modified by user as a parameter. It should be retrived from env var or static value.

File: src/rest-server/src/util/manager/user/crudK8sSecret.js (hunk 1 above)
Comment (ydye) [reply src/rest-server/src/util/manager/user/crudK8sSecret.js]: Done

File: src/rest-server/src/util/manager/user/crudK8sSecret.js (hunk 2 above)
Comment (ydye) [reply src/rest-server/src/util/manager/user/crudK8sSecret.js]: Done

**Pull request merged**